tests/invlet_test.cpp

```diff
@@ -46,6 +46,32 @@ enum test_action {
     TEST_ACTION_NUM,
 };
 
+// This is a massive hack that makes this test work without totally rewriting it after #31406
+// The number of null items in item.components is used as a UUID for the purposes of this test
+static void set_uuid( item &it, int uuid )
```

Comment:
`uuid` implies a specific type of 128-bit id.  Maybe just call it an `id` instead.

Reply:
will do